.checkstyle/checkstyle.xml

```diff
@@ -44,7 +44,7 @@
         <module name="RightCurly"/>
         <module name="EmptyStatement"/>
         <module name="ConstantName">
-            <property name="format" value="(^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$)|(^log$)"/>
+            <property name="format" value="(^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$)|(^log$)|(^loggerWrapper$)"/>
```

Comment:
Why do we needs? You seem to be changing logging everywhere. So isn't it better to strive to remove the exception than expand it?

Reply:
The final static property has to use the capitals and underscores only. Since we have an exception for `log` I thought it is a good idea to do the same exception for `loggerWrapper`. 
The `loggerWrapper` is not used everywhere. In some classes, we still use just `log` so it is not so easy to remove it from checkstyle exceptions.

Reply:
I agree with Jakub that we're better to use a proper constant name, than add another exception.

I don't like the name `logWrapper`. Something like `reconciliationLogger` would be better (though long!). But I think most of our logger should end up being "reconciliation loggers". So perhaps we use the `log` (or `LOG`) name for the reconciliation one and use some other name for the other. E.g. the logger we use when starting up the operator and not in the context of any reconciliation could be called `STARTUP_LOGGER`.